cdc/scheduler/internal/tp/scheduler.go

```diff
@@ -24,7 +24,7 @@ type scheduler interface {
 		currentTables []model.TableID,
 		aliveCaptures map[model.CaptureID]*model.CaptureInfo,
 		replications map[model.TableID]*ReplicationSet,
-	) []*scheduleTask
+		hasCaptureStopping bool) []*scheduleTask
```

Comment:
What about embedding stopping in `aliveCaptures map[model.CaptureID]*CaptureStatus`?

Reply:
I've tried it, and it works.

But `CaptureStatus` looks should be managed by `CaptureManager`. Scheduler does not need to know the existance of `CaptureStatus`.

Reply:
The recommend format is adjusted by the linter.